<?xml version="1.0" encoding="utf-8"?>
<report
	progress="">
	<activity
		result="Failed"
		duration="0ms"
		durationms="0"
		type="root"
		rid="4f3a0dcbb7f2cf2"
		totalerrorcount="1"
		totalwarningcount="0"
		totalsuccesstestcasecount="0"
		totalfailedtestcasecount="1"
		totalblockedtestcasecount="0"
		childsuccesscount="0"
		childfailedcount="0"
		childblockedcount="0"
		totalsuccesscount="0"
		totalfailedcount="0"
		totalblockedcount="0"
		user="erdogb"
		rxversion="7.2.0.0"
		host="EC168"
		osversion="Windows 7 Service Pack 1 64bit"
		runtimeversion="4.0.30319.42000"
		procarch="32bit"
		oslanguage="de-DE"
		dtnlanguage="de-CH"
		screenresolution="3840x1200"
		timestamp="20.04.2018 15:36:48"
		timestampiso="2018-04-20T13:36:48"
		endtime="20.04.2018 15:36:58"
		timeoutfactor="1"
		module="moduleAnlegen">
		<detail>
			moduleAnlegen
		</detail>
		<varbindings>
		</varbindings>
		<item
			timeRelativeToTestSuiteStartTime="00:10.432"
			timeRelativeToTestModuleStartTime="00:10.432"
			timeWallClock="15:36:58"
			time="00:10.432"
			level="Error"
			category="User"
			errimg=".\moduleAn_480cd3bd_11c973db_trace.jpg"
			errthumb=".\moduleAn_480cd3bd_11c973db_trace_thumb.jpg">
			<message>
				The test run has been aborted by the user.
			</message>
			<metainfo
				loglvl="Error">
			</metainfo>
		</item>
	</activity>
</report>
